### ISSUE TITLE
Better way to handle locksmithd?

### ISSUE THREAD

**Issue by InAnimaTe** [coreos/tectonic-forum#43]
Just noticed my single controller node broadcast a message it is going to reboot.

Was wondering if Tectonic had plans to modify this behavior for single node setups or at least make it configurable in the installer? That would be really nice, especially for those of us not deploying multi-node clusters ;)

Thanks

**Comment by philips** [comment]
Yes, this is on our roadmap to add a resource to describe Container Linux reboot policies cluster-wide.

**Comment by InAnimaTe** [comment]
Awesome! Is your roadmap something that is public? Would be cool to enable us to keep track of whats coming!
